css-basics_combined-users/css/styles.css

```diff
@@ -0,0 +1,124 @@
+/*
+  Scroll down to the bottom of the file
+  to see the instructions for this exercise.
+   ⬇
+*/
+
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+}
+
```

Comment:
add a margin with 0 and a padding with 0 to the main settings of your page so all browsers could render it similary and the elements will have a clear default setting before you add margin and padding later on

Reply:
Good point! @thisKonrad